heron/tools/tracker/src/python/handlers/metricshandler.py

```diff
@@ -145,7 +145,7 @@ def getComponentMetrics(self,
 
     if metricResponse.status.status == common_pb2.NOTOK:
       if metricResponse.status.HasField("message"):
-        Log.error(metricResponse.status.message)
+        Log.warn(metricResponse.status.message)
```

Comment:
Log.warn seems fine here, but I typically like to prepend some text to make it clear that this message is from the metrics manager. "Received the following response from the metrics manager: ..."

Same for metricstimeline.py below.